Sources/PointFree/Episode/Show.swift

```diff
@@ -71,20 +71,41 @@ private let validateUserEpisodePermission
     <<< validateCreditRequest
 
 let progressResponse: M<
-  Tuple5<
+  Tuple4<
   Either<String, Episode.Id>,
-  Int,
   Models.User?,
   SubscriberState,
-  Route?>
```

Comment:
route wasn't needed so got rid of it, and moved the `percent: Int` parameter to the end because I was able to make use of the `userEpisodePermission` middleware by having the first 3 arguments be episode, user and subscriber state.